schemas/pulse-job.yml

```diff
@@ -61,12 +61,12 @@ properties:
               This could be the organization or the individual git username
               depending on who owns the repo.
             type: "string"
-            pattern: "^[0-9a-f]+$"
+            pattern: "^[A-Za-z0-9_-]+$"
```

Comment:
ahh, good catch.  I can at least USE it.  I have to add in a couple extra characters here and there.  And a couple cases there underscore are not acceptable (``job_guid``) but other than that, this helps keep the patterns shorter.  thanks! :)